### PR TITLE
[miio] add support for viomi.waterheater.e1

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/database/viomi.waterheater.e1.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+				"actions": [
+					{
+						"command": "set_appointStart",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "appointEnd",
+				"friendlyName": "Eppoint End",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/viomi.waterheater.e1.json:130]: Is this a typo?
```suggestion
				"friendlyName": "Appoint End",
```

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/viomi.waterheater.e1.json:130]: yes, coming right to you from the chinese spec file :-)
... did not catch that one

**Review by cpmeister — APPROVED** [review]

**Comment by cpmeister** [comment]
Looks like the DCO check broke due to committing from the review suggestion.

**Comment by cpmeister** [comment]
Manually verified sign-off.

**Pull request merged**